### DISCUSSION TITLE
Website still adverting 1.0

### DISCUSSION THREAD

**Discussion by EdorianDark** [eclipse-theia/theia#8894] [General] [answered]
Even though theia is at version 1.9, on the website it still says "Learn about the 1.0 Release!"
From the website the project doesn't look very active

**Comment by vince-fugnitto** [comment, marked as answer]
@EdorianDark the `1.0.0` release was a significant milestone for the project, I'm not sure how it makes the project look inactive (since both github releases and npm show the newest versions).

Please file an issue in the appropriate repository (https://github.com/theia-ide/theia-website), and/or provide a pull-request to resolve it.
